CODEOWNERS

```diff
@@ -12,6 +12,6 @@
 # You can also use email addresses if you prefer.
 #docs/*  docs@example.com
 
-appmonitoring/* teamsword@microsoft.com
-.pipelines/azure_pipeline_mergedbranches_appmonitoring.yaml teamsword@microsoft.com
-deployment/mergebranch-webhook-deployment/* teamsword@microsoft.com
\ No newline at end of file
+appmonitoring/*                                                 @application-insights-for-aks
```

Comment:
it should be @microsoft/application-insights-for-aks

Reply:
Thank you, fixed